public/assets/css/styles.css

```
@@ -43,4 +43,13 @@ main {
 h1,
 h2 {
   @apply font-big;
+}
+
+p {
+  @apply font-small; 
+  @apply text-2xl;
```

Isn't necessary duplicate `@apply`
```suggestion
  @apply font-small text-2xl; 
```

---

:heavy_check_mark: